kernel/main.cpp

```diff
@@ -114,17 +107,13 @@ static void kernel_print_splash() {
         "\033[0m",
         VER_NAME,
         (
-            ((__DATE__)[7] - '0') * 1000 + \
-            ((__DATE__)[8] - '0') * 100  + \
-            ((__DATE__)[9] - '0') * 10   + \
-            ((__DATE__)[10] - '0') * 1     \
-        ),
-        REPO_URL
-    );
+            ((__DATE__)[7] - '0') * 1000 + ((__DATE__)[8] - '0') * 100 + ((__DATE__)[9] - '0') * 10 + ((__DATE__)[10] - '0') * 1),
```

Comment:
I know this isn't your fault -- I need to configure the formatter a bit better. Could you revert this change? 😬

Reply:
No worries, fixed it.  I thought I saw some stuff shift that wasn't mine.  :)